incubator/nifi/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: nifi
+home: https://nifi.apache.org/
+version: 0.0.1
+appVersion: 1.6.0
+description: A Helm chart built to install NiFI on a K8S cluster
+keywords:
+- analytics
+- data
+sources:
+- https://github.com/apache/nifi
+maintainers:
+- name: William Hearn
```

Comment:
Can you update this to a valid github username? We require that to be able to contact folks.